### PR TITLE
Add buttons & metadata changes

### PR DESCRIPTION
**Pull request by KurokuTetsuya**

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Bump version, upload proof of it working

**Comment by KurokuTetsuya** [comment]
Proof it works:
image
image

File: websites/O/Otakudesu/dist/metadata.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
   "description": {
     "en": "Download, watch and stream anime with Indonesian subtitle and with 240p, 360p, 480p, & 720p resolution with MP4 format & MKV along with the batch."
   },
-  "url": "otakudesu.tv",
-  "version": "1.0.1",
+  "url": "otakudesu.moe",
+  "version": "1.1.1",
```
Comment (Bas950) [inline websites/O/Otakudesu/dist/metadata.json:11]: 1.1.0 or 1.0.2

**Review by Bas950 — CHANGES_REQUESTED** [review]
Bump version, upload proof of it working

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**